packages/card/src/react/__stories__/index.story.tsx

```diff
@@ -286,8 +286,8 @@ storiesOf('fullOverlay', module)
 
 const sizeStory = storiesOf('size', module)
 Object.values(Card.sizes).forEach(size =>
-  sizeStory.add(size, _ =>
-    ConstrainSizeDecorator(size)(_ => (
+  sizeStory.add(size, () =>
+    ConstrainSizeDecorator(size)(() => (
```

Comment:
fyi there are newer ways to wrap decorators around stories. not really a big deal because all our stories use the old syntax anyway...i see some more refactoring in our future